.gitignore

```diff
@@ -44,6 +44,7 @@ htmlcov
 .idea
 org.eclipse.core.resources.prefs
 .vscode
+.vs
```

Comment:
Just curious: what would this file be?

Reply:
I was using Visual Studio and it created this folder. This is VS internal bookkeeping of the project.

Reply:
thx